src/main/java/com/learnjava/parallelstreams/ParallelStreamPerformance.java

```diff
@@ -19,6 +18,7 @@ public int sum_using_intstream(int count, boolean isParallel){
         int sum = intStream
                 .sum();
         timeTaken();
+        stopWatchReset();
```

Comment:
@adityagarde The startTimer() function already calls the `stopWatchReset()`

```
    public static void startTimer(){
        stopWatchReset();
        stopWatch.start();
    }
```

Reply:
@dilipsundarraj1 I had not included the `stopWatchReset();` in the `startTimer()` method. This will surely work then, Thanks! Sorry for the trouble!

Reply:
No Problem!